build/build-image/Dockerfile

```diff
@@ -123,6 +123,11 @@ RUN mkdir /tmp/hugo && \
 RUN curl -sL https://deb.nodesource.com/setup_11.x | bash - && \
     apt-get install -y nodejs
 
+# install API reference docs generator
+RUN mkdir -p /go/src/github.com/ahmetb && \
+    cd /go/src/github.com/ahmetb && git clone https://github.com/ahmetb/gen-crd-api-reference-docs && \
```

Comment:
I think you should probably pin to a commit here.
Given there's a copy of the template in this repo, I don't want to accidentally break the downstream.

Reply:
Thanks @ahmetb for the review. I was thinking about that, will provide an update in a separate PR.